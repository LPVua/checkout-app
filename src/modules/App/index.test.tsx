import { App } from ".";
import * as React from "react";
import { render, cleanup } from "@testing-library/react";

describe("App", () => {
  afterEach(cleanup);

  it("should render", () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});
