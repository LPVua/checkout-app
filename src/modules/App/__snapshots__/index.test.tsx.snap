// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
<DocumentFragment>
  .emotion-67 {
  padding: .8rem;
}

.emotion-23 {
  margin-bottom: 1.2rem;
}

.emotion-0 {
  line-height: 22px;
  font-size: 1.8rem;
  margin-bottom: 1.2rem;
}

.emotion-1 {
  position: absolute;
  left: 1.8rem;
  top: 1.2rem;
  width: auto;
  color: #95a5a6;
  -webkit-transform: translateY(0.2rem) scale(1);
  -ms-transform: translateY(0.2rem) scale(1);
  transform: translateY(0.2rem) scale(1);
  -webkit-transform-origin: top left;
  -ms-transform-origin: top left;
  transform-origin: top left;
  -webkit-transition: -webkit-transform .15s linear;
  -webkit-transition: transform .15s linear;
  transition: transform .15s linear;
  z-index: 0;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  color: var(--dark-1);
  z-index: 1;
}

.emotion-10 {
  width: 3.6rem;
  height: 3.6rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: .6rem;
}

.emotion-27 {
  font-size: 1.4rem;
  padding-left: .6rem;
}

.emotion-32 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-31 {
  width: 100%;
}

.emotion-29 {
  font-weight: bold;
  margin-bottom: .3rem;
}

.emotion-30 {
  font-size: 1.4rem;
  color: var(--grey-2);
}

.emotion-33 {
  margin-top: .6rem;
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  position: relative;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-height: 4.8rem;
  border: .1rem solid var(--grey-1);
  background: white;
  border-radius: .3rem;
  margin-bottom: .4rem;
}

.emotion-2 {
  padding: .6rem;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  width: 100%;
  min-width: 0;
  height: 100%;
  background: transparent;
  padding-left: 1.8rem;
  border-radius: .3rem;
  border: none;
  margin: 0;
  font: inherit;
  outline: none;
  padding-top: 1.8rem;
}

.emotion-9 {
  position: absolute;
  left: 1.8rem;
  top: 1.2rem;
  width: auto;
  color: #95a5a6;
  -webkit-transform: translateY(0.2rem) scale(1);
  -ms-transform: translateY(0.2rem) scale(1);
  transform: translateY(0.2rem) scale(1);
  -webkit-transform-origin: top left;
  -ms-transform-origin: top left;
  transform-origin: top left;
  -webkit-transition: -webkit-transform .15s linear;
  -webkit-transition: transform .15s linear;
  transition: transform .15s linear;
  z-index: 0;
  -webkit-transform: translateY(-0.6rem) scale(.75);
  -ms-transform: translateY(-0.6rem) scale(.75);
  transform: translateY(-0.6rem) scale(.75);
}

.emotion-15 {
  padding: 0 1.8rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: .6rem;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  width: 100%;
  min-width: 0;
  height: 100%;
  background: transparent;
  padding-left: 1.8rem;
  border-radius: .3rem;
  border: none;
  margin: 0;
  font: inherit;
  outline: none;
  padding-top: 1.8rem;
}

.emotion-14 {
  width: 3.6rem;
  height: 3.6rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: .6rem;
  background: var(--blue-1);
  color: var(--white-1);
  border-radius: 3.6rem;
}

.emotion-19 {
  padding: .6rem;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  width: 100%;
  min-width: 0;
  height: 100%;
  background: transparent;
  padding-left: 1.8rem;
  border-radius: .3rem;
  border: none;
  margin: 0;
  font: inherit;
  outline: none;
  padding-top: 1.8rem;
  resize: none;
}

.emotion-22 {
  height: 4.8rem;
  border: none;
  background: var(--blue-1);
  color: var(--white-1);
  font-size: inherit;
  border-radius: .3rem;
  padding: 0 .8rem;
  margin: 0;
  outline: none;
  -webkit-transition: background .3s linear;
  transition: background .3s linear;
  pointer-events: none;
  opacity: .5;
}

.emotion-22:active {
  background: var(--blue-2);
}

.emotion-34 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-bottom: 1.2rem;
}

.emotion-28 {
  height: 36px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-size: 1.8rem;
  padding: 0 0.6rem;
  border-radius: .3rem;
  background: #1abc9c;
  color: white;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  background: var(--red-1);
  margin-right: .6rem;
}

.emotion-36 {
  height: 36px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-size: 1.8rem;
  padding: 0 0.6rem;
  border-radius: .3rem;
  background: #1abc9c;
  color: white;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  background: #f1c40f;
  margin-right: .6rem;
}

.emotion-52 {
  height: 36px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-size: 1.8rem;
  padding: 0 0.6rem;
  border-radius: .3rem;
  background: #1abc9c;
  color: white;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  background: #1abc9c;
  margin-right: .6rem;
}

<div
    class="emotion-67"
  >
    <div
      class="emotion-0"
    >
      Add Review
    </div>
    <div
      class="emotion-23"
    >
      <div
        class="emotion-4"
        data-testid="name-input"
      >
        <div
          class="emotion-1"
        >
          Name
        </div>
        <div
          class="emotion-3"
        >
          <input
            class="emotion-2"
            data-testid="input"
            value=""
          />
        </div>
      </div>
      <div
        class="emotion-4"
        data-testid="email-input"
      >
        <div
          class="emotion-1"
        >
          Email
        </div>
        <div
          class="emotion-3"
        >
          <input
            class="emotion-2"
            data-testid="input"
            value=""
          />
        </div>
      </div>
      <div
        class="emotion-4"
        data-testid="rating-input"
      >
        <div
          class="emotion-9"
        >
          Rating
        </div>
        <div
          class="emotion-3"
        >
          <div
            class="emotion-15"
          >
            <div
              class="emotion-10"
              data-testid="rating-1"
            >
              1
            </div>
            <div
              class="emotion-10"
              data-testid="rating-2"
            >
              2
            </div>
            <div
              class="emotion-10"
              data-testid="rating-3"
            >
              3
            </div>
            <div
              class="emotion-10"
              data-testid="rating-4"
            >
              4
            </div>
            <div
              class="emotion-14"
              data-testid="rating-5"
            >
              5
            </div>
          </div>
        </div>
      </div>
      <div
        class="emotion-4"
        data-testid="comments-input"
      >
        <div
          class="emotion-1"
        >
          Comments
        </div>
        <div
          class="emotion-3"
        >
          <textarea
            class="emotion-19"
            data-testid="input"
            rows="6"
          />
        </div>
      </div>
      <button
        class="emotion-22"
        data-testid="submit-button"
      >
        Add Review
      </button>
    </div>
    <div
      class="emotion-0"
    >
      Happiness Trend
    </div>
    <div
      class="recharts-responsive-container emotion-23"
      style="width: 100%; height: 200px;"
    >
      <div
        style="position: absolute; width: 0px; height: 0px; visibility: hidden; display: none;"
      />
    </div>
    <div
      class="emotion-0"
    >
      Latest Reviews
    </div>
    <div
      class="emotion-34"
    >
      <div
        class="emotion-32"
      >
        <div
          class="emotion-28"
        >
          2 / 
          <span
            class="emotion-27"
          >
            5
          </span>
        </div>
        <div
          class="emotion-31"
        >
          <div
            class="emotion-29"
          >
            Pavlo Lompas
          </div>
          <div
            class="emotion-30"
          >
            paullompas@gmail.com
          </div>
        </div>
      </div>
      <div
        class="emotion-33"
      >
        Amazing Service!
      </div>
    </div>
    <div
      class="emotion-34"
    >
      <div
        class="emotion-32"
      >
        <div
          class="emotion-36"
        >
          3 / 
          <span
            class="emotion-27"
          >
            5
          </span>
        </div>
        <div
          class="emotion-31"
        >
          <div
            class="emotion-29"
          >
            Pavlo Lompas
          </div>
          <div
            class="emotion-30"
          >
            paullompas@gmail.com
          </div>
        </div>
      </div>
      <div
        class="emotion-33"
      >
        Amazing Service!
      </div>
    </div>
    <div
      class="emotion-34"
    >
      <div
        class="emotion-32"
      >
        <div
          class="emotion-28"
        >
          1 / 
          <span
            class="emotion-27"
          >
            5
          </span>
        </div>
        <div
          class="emotion-31"
        >
          <div
            class="emotion-29"
          >
            Pavlo Lompas
          </div>
          <div
            class="emotion-30"
          >
            paullompas@gmail.com
          </div>
        </div>
      </div>
      <div
        class="emotion-33"
      >
        Amazing Service!
      </div>
    </div>
    <div
      class="emotion-34"
    >
      <div
        class="emotion-32"
      >
        <div
          class="emotion-52"
        >
          4 / 
          <span
            class="emotion-27"
          >
            5
          </span>
        </div>
        <div
          class="emotion-31"
        >
          <div
            class="emotion-29"
          >
            Pavlo Lompas
          </div>
          <div
            class="emotion-30"
          >
            paullompas@gmail.com
          </div>
        </div>
      </div>
      <div
        class="emotion-33"
      >
        Amazing Service!
      </div>
    </div>
    <div
      class="emotion-34"
    >
      <div
        class="emotion-32"
      >
        <div
          class="emotion-52"
        >
          5 / 
          <span
            class="emotion-27"
          >
            5
          </span>
        </div>
        <div
          class="emotion-31"
        >
          <div
            class="emotion-29"
          >
            Pavlo Lompas
          </div>
          <div
            class="emotion-30"
          >
            paullompas@gmail.com
          </div>
        </div>
      </div>
      <div
        class="emotion-33"
      >
        Amazing Service!
      </div>
    </div>
  </div>
</DocumentFragment>
`;
